.github/workflows/runFrontendTests.yml

```
@@ -32,6 +32,7 @@ jobs:
           node-version-file: frontend/package.json
           cache: 'npm'
           cache-dependency-path: frontend/package-lock.json
+          enable-corepack: true
```

actionlint (1.7.12)

[error] input "enable-corepack" is not defined in action "actions/setup-node@v6". available inputs are "architecture", "cache", "cache-dependency-path", "check-latest", "mirror", "mirror-token", "node-version", "node-version-file", "package-manager-cache", "registry-url", "scope", "token"

(action)